auto_nag/scripts/to_triage.py

```diff
@@ -73,13 +73,38 @@ def get_bz_params(self, date):
             "include_fields": fields,
             "product": list(prods),
             "component": list(comps),
-            "resolution": "---",
-            "f1": "priority",
+            "email1": "intermittent-bug-filer@mozilla.bugs",
```

Comment:
Could you order the fields to make them easier to read ?
For example:
```
f1: ...
o1: ...
v1: ...
```
It's a way easier to understand what the query is doing instead of having to jump all the time/